src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -154,6 +154,23 @@ const debounceFrame = (func: DebouncableFn): DebouncedFn => {
 	};
 };
 
+const createIconElements = (): {[key: string]: ReactElement<any>} => {
+	const iconList = ['vm', 'cloud', 'tag', 'inbox', 'check', 'loading', 'warning'];
+	const elementLibrary: {[key: string]: ReactElement<any>} = {};
+	iconList.forEach(iconKey => {
+		elementLibrary[iconKey] = createElement('span', {className: `codicon codicon-${iconKey}`});
+	});
+	return elementLibrary;
+};
+
+const iconElementLibrary = createIconElements();
+
+const getIconElementLibrary = (): (key: string) => ReactElement => {
+	return (iconKey: string) => {
```

Comment:
const getIconElementLibrary = (): (key: string) => ReactElement => {
```suggestion
	const getIconElementLibrary = (iconKey: string): ReactElement => {
			return iconElementLibrary[iconKey];
	}
```

and then in the render side of the GraphWrapper.jsx:

```
<GraphContainer
  ...
  getExternalIcon={getIconElementLibrary}
/>
```